webpack.config.js

```diff
@@ -44,7 +44,7 @@ module.exports = {
     new webpack.DefinePlugin({
       'process.env.NODE_ENV': JSON.stringify('staging')
     }),
-    new DashboardPlugin({ port: 3001 })
+    new DashboardPlugin({ port: 3001 }) // Change this here and in the package.json start script if needed
```

Comment:
I mean change the port number if needed. If you need to, it has to be changed in both places. Is there a better way I can word this comment?